doc/src/manual/performance-tips.md

```diff
@@ -13,7 +13,7 @@ The functions should take arguments, instead of operating directly on global var
 
 ## Avoid untyped global variables
 
-An untyped global variable might have its value, and therefore possibly its type, changed at any point. This makes
+The value of an untyped global variable might change at any point, possibly leading to a change of its type. This makes
```

Comment:
How about "Untyped global variables might change value or type at any time."

Reply:
I think this makes it sound like type change is independent of value change.